### PR TITLE
Update local and remote docs

### LINKED ISSUE #501: Refine and test setup instructions for Windows
**As a** Windows user,
**I want to** have clear instructions on how to setup BLS Wallet with WSL2 on a windows machine
**So that** I can run BLS Wallet components with WSL2 easily

Following #500, we should test setting up the BLS Wallet repo on a windows machine using WSL2. After doing this successfully, we should amend the instructions if needed with any additional detail or gotchas.

**Note if you're developing on a linux/mac. You could test this by using a Windows VM and downloading WSL2**

### Acceptance criteria
- Can follow the [local development](https://github.com/web3well/bls-wallet/blob/main/docs/local_development.md) instructions to completion on a Windows machine using WSL2. Completion means running a node, a local aggregator and sending a transaction successfully with Quill.
- Can follow the [remote development](https://github.com/web3well/bls-wallet/blob/main/docs/remote_development.md) instructions to completion on a Windows machine using WSL2. Completion means running a local aggregator pointing at testnet (arbitrum goerli) and sending a transaction successfully with Quill on testnet (arbitrum goerli)
- Setup instructions for [local development](https://github.com/web3well/bls-wallet/blob/main/docs/local_development.md) and [remote development](https://github.com/web3well/bls-wallet/blob/main/docs/remote_development.md) are updated with any additional detail or gotchas.

### PR DESCRIPTION
**Pull request by ri-dev-git**
## What is this PR doing?
Addition of setup steps for #501.

## Does this PR resolve or contribute to any issues?
yes, #501

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: docs/local_development.md [hunk 1]
```diff
@@ -83,6 +87,7 @@ yarn run dev:chrome # or dev:firefox, dev:opera
 1. Go to Chrome's [extension page](chrome://extensions).
 2. Enable `Developer mode`.
 3. Either click `Load unpacked extension...` and select `./extension/extension/chrome` or drag that folder into the page.
+4. Transfer eth from metamask to Quill if using `yarn start-hardhat` in `./contracts`.
```
Comment (JohnGuilding) [inline docs/local_development.md]:
```suggestion
4. Transfer ETH from a funded account to Quill - e.g. an imported MetaMask account
```

You can fund the account in multiple ways beyond metamask, for example, a script. So could make the suggestion broader and add metamask as an example.

File: docs/local_development.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ cd ./aggregator
 cp .env.local.example .env
 ```
 
+> **Note: The .env file in the aggregator needs to be updated everytime the network changes from local to remote and vice-versa.**
+
+> **.env.local.example** for **local**
```
Comment (JohnGuilding) [inline docs/local_development.md]: > **.env.local.example** for **local**

I'd say you could remove this line, as it is stated in the code example above. Feel free to disagree!

File: docs/local_development.md [hunk 3]
```diff
@@ -83,6 +87,7 @@ yarn run dev:chrome # or dev:firefox, dev:opera
 1. Go to Chrome's [extension page](chrome://extensions).
 2. Enable `Developer mode`.
 3. Either click `Load unpacked extension...` and select `./extension/extension/chrome` or drag that folder into the page.
+4. Transfer eth from metamask to Quill if using `yarn start-hardhat` in `./contracts`. 
 
 ### Firefox
```
Comment (JohnGuilding) [inline docs/local_development.md:90]: Could you add the same point to the firefox instructions too?

e.g. `4. Transfer ETH from a funded account to Quill - e.g. an imported MetaMask account`

File: docs/remote_development.md [hunk 4]
```diff
@@ -94,6 +94,10 @@ PRIVATE_KEY_ADMIN=PK1
 ...
 ```
 
+> **Note: The .env file in the aggregator needs to be updated everytime the network changes from local to remote and vice-versa.**
+
+> **.env.example** for **remote**
```
Comment (JohnGuilding) [inline docs/remote_development.md]: Same comment as above, could remove this line. 

I can see that there is no code example here for copying .env.example. Could you add this snippet above like this?

___
_The [aggregator](../aggregator/) is a service that accepts transaction bundles (including those that contain a single transaction) and submits aggregations of these bundles to L2.

```sh
cd ./aggregator
cp .env.example .env
```


Update the following values in `./aggregator/.env`
___

**Review by JohnGuilding — CHANGES_REQUESTED** [review]
Thanks for doing this @ri-dev-git!

Left a few suggestions

**Comment by ri-dev-git** [comment]
Hi @JohnGuilding! I have made the changes please checkI

File: docs/remote_development.md [hunk 5]
```diff
@@ -83,7 +83,14 @@ This file can be committed so others can use your deployed contracts.
 
 The [aggregator](../aggregator/) is a service that accepts transaction bundles (including those that contain a single transaction) and submits aggregations of these bundles to L2.
 
-Update the following values in `./aggregator/.env`
+```
+cd ./aggregator
+cp .env.example .env
+```
+> **Note: This .env file now uses `arbitrum` RPCs.**
```
Comment (JohnGuilding) [inline docs/remote_development.md]: Could you remove this comment and the whitespace below it? Don't think it's needed as arbitrum is mentioned above and you can see the arbitrum goerli rpcs in the .env file

**Comment by JohnGuilding** [comment]
@ri-dev-git one final comment then we're ready to merge!

**Comment by ri-dev-git** [comment]
HI @JohnGuilding! Done with it!

**Review by JohnGuilding — APPROVED** [review]
Looks good! Awesome thanks for doing this @ri-dev-git

**Comment by JohnGuilding** [comment]
@ri-dev-git now that you've got your windows setup sorted, feel free to pick up another issue if you want to start writing  some code, as I know you haven't done much of that yet 🙂

Check out issues marked as [good first issue](https://github.com/web3well/bls-wallet/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)

**Pull request merged**